rpc/src/v1/informant.rs

```diff
@@ -135,7 +135,7 @@ impl<T: Default + Copy + Ord> StatsCalculator<T> {
 #[derive(Default, Debug)]
 pub struct RpcStats {
 	requests: RwLock<RateCalculator>,
-	roundtrips: RwLock<StatsCalculator<u32>>,
+	roundtrips: RwLock<StatsCalculator<u128>>,
```

Comment:
What Clippy lint does this change correspond to?

Reply:
That's a good question it was this one https://rust-lang.github.io/rust-clippy/master/#duration_subsec

But I replaced our `own` as_micros() method with `Duration::as_micros` and don't truncate the result as `u32`